tst/EnergyPlus/unit/HVACUnitaryBypassVAV.unit.cc

```diff
@@ -777,6 +777,7 @@ TEST_F(CBVAVSys, UnitaryBypassVAV_NoOASys)
     DataLoopNode::Node(MixedAir::OAMixer(1).InletNode).Enthalpy = 71299.267;
 
     DataLoopNode::Node(cbvav.CBVAVBoxOutletNode(1)).MassFlowRateMax = 0.61;
+    DataLoopNode::Node(cbvav.CBVAVBoxOutletNode(1)).MassFlowRate = 0.61;
```

Comment:
Now that BypassMassFlowFrac is only based on box flow, each unit test needed to include the box node flow rate.